### PR TITLE
Linq: Support Contains() with composite clustering keys

### PR DESCRIPTION
**Pull request by jorgebay**
Example:

```csharp
// SELECT id3, id2, id1 FROM tbl1 WHERE id1 = ? AND (id2, id3) IN ?
table.Where(t => t.Id1 == id && list.Contains(Tuple.Create(t.Id2, t.Id2)))
     .Execute();
```

### CODE REVIEW

File: src/Cassandra.IntegrationTests/Linq/LinqMethods/Where.cs [hunk 1]
```diff
@@ -178,6 +180,133 @@ public void LinqWhere_AppendMultipleTimes()
...
+                TimeColumn = 1
+            }).Execute();
+            var localList = new List<int> {0, 2};
+            var emptyResults = table.Where(t => t.UserId == 1 && localList.Contains(t.Date)).Execute();
+            Assert.NotNull(emptyResults);
+            Assert.AreEqual(0, emptyResults.ToArray().Length);
+            
+            localList.Add(1); //adding to list existent tuple
+            var tCreateResults = table.Where(t => t.UserId == 1 && localList.Contains(t.Date)).Execute();
+            Assert.NotNull(tCreateResults, "Linq Where using 'Tuble.Create': Result should not be null");
```
Comment (jorgebay) [inline src/Cassandra.IntegrationTests/Linq/LinqMethods/Where.cs]: I think the error message on the assertion is not correct.

File: src/Cassandra.IntegrationTests/Linq/LinqMethods/Where.cs [hunk 2]
```diff
@@ -178,6 +180,133 @@ public void LinqWhere_AppendMultipleTimes()
...
+            var tNewResultObj = tNewResultsArr[0];
+            Assert.AreEqual(1, tNewResultObj.UserId);
+            Assert.AreEqual(1, tNewResultObj.Date);
+            Assert.AreEqual(1, tNewResultObj.UserId);
+
+            //invalid case: string.Contains
+            Assert.Throws<InvalidOperationException>(() =>
+                table.Where(t => t.UserId == 1 && "error".Contains($"{t.Date}")).Execute());
+        }
+        [Test]
```
Comment (jorgebay) [inline src/Cassandra.IntegrationTests/Linq/LinqMethods/Where.cs]: Missing line break :)

File: src/Cassandra.IntegrationTests/Linq/LinqMethods/Where.cs [hunk 3]
```diff
@@ -178,6 +180,133 @@ public void LinqWhere_AppendMultipleTimes()
...
+            Assert.NotNull(tCreateResults, "Linq Where using 'Tuble.Create': Result should not be null");
+            var tCreateResultsArr = tCreateResults.ToArray();
+            Assert.AreEqual(1, tCreateResultsArr.Length);
+            var tCreateResultObj = tCreateResultsArr[0];
+            Assert.AreEqual(1, tCreateResultObj.UserId);
+            Assert.AreEqual(1, tCreateResultObj.Date);
+            Assert.AreEqual(1, tCreateResultObj.UserId);
+            
+            var tNewResults = table.Where(t => t.UserId == 1 && localList.Contains(t.Date)).Execute();
+            Assert.NotNull(tNewResults, "Linq Where using 'new Tuple()': Result should not be null");
```
Comment (jorgebay) [inline src/Cassandra.IntegrationTests/Linq/LinqMethods/Where.cs]: Incorrect error message.

**Comment by landim** [comment]
Removed the assertion message (not needed). And removed a duplicated code.

**Pull request merged**